js/00-main.js

```diff
@@ -1,25 +1,62 @@
-jQuery( function () {
-	const INTERNAL_EVENTS_CATEGORY = 4;
+document.addEventListener("DOMContentLoaded", function () {
+	// Check if we are in the room booking page ('/rooms/book') as it is not compatible with JQuery
+	const currentUrl = window.location.pathname;
+	if (currentUrl != "/rooms/book" && currentUrl != "/rooms") {
+		jQuery(function () {
+			const INTERNAL_EVENTS_CATEGORY = 4;
 
-	// moves the menu header
-	jQuery('.toolbar.global-menu').detach().appendTo('div.page-header>div>a');
+			// moves the menu header
+			const $toolbar = jQuery('.toolbar.global-menu').detach();
+			const $targetDiv = jQuery('div.page-header>div');
+			const $logo = $targetDiv.find('a').first();
+			$toolbar.insertAfter($logo);
 
-	// display moved header
-	jQuery('div.page-header').css({display: 'block'});
+			// update and remove some navigation items
+			$toolbar.find('a, button').each(function () {
+				switch (jQuery(this).text().toLowerCase().trim().replace(' ', '_')) {
+					case 'home':
+					case 'create_event':
+					case 'my_profile':
+					case 'room_booking':
+						jQuery(this).closest('li').remove();
+						break;
+					case 'administration':
+						jQuery(this).text('Admin settings');
+					default:
+						break;
+				}
+			});
```

Comment:
You might be able to do this in CSS by using

```
#global-menu > menu > li > a[href="/"],
#global-menu > menu > li > ind-menu,
#global-menu > menu > li > a[href="/rooms/"],
#global-menu > menu > li > a[href="/user/dashboard/"] {
  display: none !important;
}
```

That won't change admin settings, and if another menu comes along it would hide that as well, but I think Administration is fine.